src/sql/workbench/browser/designer/media/designer.css

```diff
@@ -101,4 +101,26 @@
 .designer-component .components-grid .full-row.group-header {
 	font-weight: bold;
 	line-height: 25px;
+	overflow-y: auto;
+}
+
+.designer-component .description-component {
+	margin: 15px;
+	height: 90px;
```

Comment:
can we use `flex: 0 0 auto;` instead of having fixed height?

Reply:
do you want the description to be sticking to the bottom? I can change it, but it looks better with some height

Reply:
sounds good.